### ISSUE TITLE
Feature: Providing access to data/unifi/data via SSH

### ISSUE THREAD

**Issue by boesing** [hassio-addons/addon-unifi#216]
# Problem/Motivation

> I would say I am a more advanced user of the unifi stuff and thus had to make changes to my USG using the `data/unifi/data/sites/*` files to have a persistent configuration. 

## Expected behavior

> Having access via the SSH addon, e.g. via `share/unfii/sites` so I can modify these files in a more convenient way. I'd be fine with having a dedicated feature flag for that, so it is not enabled by default for example.

## Actual behavior

> I have a `config.gateway.json` which I actually have to modify via SSH into the host, starting a shell in the unifi docker container and then change to the appropriate directory to create/change the `config.gateway.json`. Due to the lack of file editors, I have to copy the contents of that file, paste it to an editor, modify it as needed and then having to paste it back into that file ( `cat` to the rescue).

## Proposed changes

> Mount `share` to the host system
> Mount `data/share/sites` to `share/unifi` (I think symlinking wont work here 😅)
> Make that feature opt-in via configuration (optional)

If I can be of any help, just point me in the right direction. I am willing to contribute this if thats a feature you think this addon might benefit from.

**Comment by frenck** [comment]
If you need that, this add-on is not for you.

**Comment by boesing** [comment]
bold statement.

**Comment by frenck** [comment]
No it actually has been discussed quite a few times before. If you took the time to look into the history, you would have found previous discussions.

Thanks for the downvote, motivating 👍

**Comment by boesing** [comment]
Sounds like you had a bad day. 
you are right, I did not check existing issues and obviously that happens a lot and I guess that this is due to the fact that there is not documented that such advanced configurations are not used by the average user. I am sorry that I wasted your time.

the downvote is for the passive aggressive statement, no need to take it Personal. that had nothing to do with your work, which is awesome.
